extensions/aimlab/src/components/Creator.tsx

```diff
@@ -0,0 +1,49 @@
+import { List, ActionPanel, Action, Icon, useNavigation, Color } from "@raycast/api";
+import { Author } from "../types/author.types";
+import generateCreatorAccessories from "../utils/generateCreatorAccessories";
+import UserProfile from "../views/profile";
+
+type PropTypes = {
+  author: Author;
+  index: number;
+};
+
+const rankColors = [Color.Yellow, Color.SecondaryText, Color.Orange];
+
+const AuthorComponent = ({ author, index }: PropTypes) => {
+  const { push } = useNavigation();
+  const rank = index + 1;
+
+  return (
+    <List.Item
+      icon={
+        rank <= 3
+          ? {
+              value: { source: Icon.Crown, tintColor: rankColors[rank - 1] },
+              tooltip: "Rank: " + rank,
+            }
+          : ""
+      }
+      title={author.username}
+      subtitle={{ value: "#" + rank, tooltip: "Rank" }}
+      accessories={generateCreatorAccessories(author)}
+      keywords={[author.username]}
+      actions={
+        <ActionPanel title="Top Creator Actions">
+          <Action
+            title="Show Player Details"
+            icon={Icon.ArrowRight}
+            onAction={() => push(<UserProfile username={author.username} />)}
+          />
```

Comment:
`push` is designed for programmatic navigation. You can use `Action.Push` instead.